### DISCUSSION TITLE
Modifying the direction of chevron in split button

### DISCUSSION THREAD

**Discussion by ngaluen** [microsoft/fluentui#29850] [Q&A] [answered]
Hello, I am quite new to the developer community. Sorry if I raised a stupid question but I wonder if it is possible to modify the direction of the chevron of split button? Now it is set facing downwards in default, but I would like to make it point to the right.

The component that I'm referring (and I am using React):
https://fluent2.microsoft.design/components/web/react/button/usage

**Comment by khmakoto** [comment, marked as answer]
Hey, you can easily do this by making use of our `slot` APIs. An example of how to do this is [in this codesandbox](https://codesandbox.io/s/splitbutton-with-chevron-right-menu-icon-2p64fz).

![image](https://github.com/microsoft/fluentui/assets/7798177/245e5db6-fad5-4ba8-8383-bee32cbf5f3d)

**Reply by ngaluen** [reply]
Thank you so much!!! I did try to modify the menuButton but I did not know about changing the menuIcon and the slot feature. I will look it up, thank you!

**Reply by khmakoto** [reply]
No problem!
